app/assets/javascripts/solidus_paypal_braintree/client.js

```diff
@@ -262,20 +263,20 @@ SolidusPaypalBraintree.Client.prototype.initializeApplePaySession = function(con
   session.begin();
 },
 
-SolidusPaypalBraintree.Client.prototype._buildTransaction = function(payload, config, shippingContact) {
+SolidusPaypalBraintree.Client.buildTransaction = function(payload, config, shippingContact) {
```

Comment:
It made it easier to reference them from within `session.onpaymentauthorized`. But yes, this was intended to be a temporary step while I refactored other code. On my list to fix up before I remove the '[WIP]' from the title.